operators/hack/deployer/runner/settings.go

```diff
@@ -57,3 +63,24 @@ type RunConfig struct {
 	Id        string                 `yaml:"id"`
 	Overrides map[string]interface{} `yaml:"overrides"`
 }
+
+func ParseFiles(plansFile, runConfigFile string) (plans Plans, runConfig RunConfig, err error){
+	var yml []byte
+	yml, err = ioutil.ReadFile(plansFile)
+	if err == nil {
```

Comment:
If `plansFile` does not exist the error is not reported.
```suggestion
	if err != nil {
		return
	}
	err = yaml.Unmarshal(yml, &plans)
	if err != nil {
		return
	}
```